HousingRepairsSchedulingApi/Controllers/AppointmentsController.cs

```diff
@@ -1,11 +1,10 @@
 namespace HousingRepairsSchedulingApi.Controllers
 {
     using System;
-    using System.Text.Json;
+    using System.Collections.Generic;
```

Comment:
This `using` wasn't required before and there are no code changes to this file - is it required?

Reply:
there is a change in this file